utils.py

```diff
@@ -76,13 +109,40 @@ def dataset_from_folder(folder, img_ext='.png', mask_ext='.trn'):
         yield load_image_for_dataset(file)
 
 if __name__ == '__main__':
-    for datum in dataset_from_folder('./plzen'):
+    data, data_mask = load_dataset('./plzen')
+    eigenvalues, pc = calc_PCA(data)
+    jitter_data = ra.add_color_noise(data, eigenvalues, pc)
+    pca_data = np.clip(jitter_data * 255, 0, 255).astype('uint8')
+
+    flipped_data, flipped_mask = ra.flip_batch(data, data_mask)
```

Comment:
@mrshu I think this is better because its flips entire batch at the same time and numpy takes care of all the 'loop' stuff which is faster then doing this sequentially. Also I think we should stick to batches because they going to be loaded to memory and if we would do it on entire dataset we might run into memory problems latter on.